assets/js/data/cart/actions.js

```diff
@@ -125,3 +125,27 @@ export function* removeCoupon( couponCode ) {
 
 	yield receiveRemovingCoupon( '' );
 }
+
+/**
+ * Selects a shipping rate.
+ *
+ * @param {Array} shippingRates the shipping rates ids
+ */
+export function* selectShippingRate( shippingRates ) {
```

Comment:
Didn’t think I need to add `isApplyingShippingRate` like coupons, but might do it if it is required.